src/manifold/src/manifold.cpp

```diff
@@ -192,7 +192,8 @@ MeshGL Manifold::GetMeshGL(ivec3 normalIdx) const {
       !isOriginal && glm::all(glm::greaterThan(normalIdx, ivec3(2)));
 
   MeshGL out;
-  out.precision = Precision();
+  out.precision = std::max(Precision(), std::numeric_limits<float>::epsilon() *
+                                            BoundingBox().Scale());
```

Comment:
Here's my update to make the `MeshGL` tests pass - just account for the additional rounding error. Now we're back to `kTolerance = 1e-12` and everything passes on my machine.